epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -54,6 +61,31 @@ public void bindChanges(@NonNull ColorHolder holder,
     }
   }
 
+  @Nullable
+  @Override
+  public EpoxyItemAnimation bindChangesAnimated(@NonNull ColorHolder view,
+                                                @NonNull EpoxyModel<?> previouslyBoundModel,
+                                                @Nullable EpoxyItemAnimation interruptedAnimation) {
+
+    int currentColor = ((ColorModel) previouslyBoundModel).color;
+    float currentRotation = 0f;
+
+    if(interruptedAnimation != null) {
+      AnimatorSet previousAnim = (AnimatorSet) ((EpoxyItemAnimation.AnimatorWrapper) interruptedAnimation).wrappedAnimator;
```

Comment:
You're right, it does seem like it could be redundant in a lot of cases. I wonder if we would be limiting people by not providing the previous model though, or encouraging bad patterns by referencing the viewholder instead of the model.